sdk/keyvault/keyvault-admin/review/keyvault-admin.api.md

```diff
@@ -0,0 +1,102 @@
+## API Report File for "@azure/keyvault-admin"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import * as coreHttp from '@azure/core-http';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { TokenCredential } from '@azure/core-http';
+
+// @public (undocumented)
+export class AccessControlClient {
+    constructor(vaultUrl: string, credential: TokenCredential, pipelineOptions?: AccessControlClientOptions);
+    createRoleAssignment(scope: RoleAssignmentScope, name: string, options?: CreateRoleAssignmentOptions): Promise<KeyVaultRoleAssignment>;
```

Comment:
I have updated this by this point, to be:

```ts
    createRoleAssignment(scope: RoleAssignmentScope, name: string, roleDefinitionId: string, principalId: string, options?: CreateRoleAssignmentOptions): Promise<KeyVaultRoleAssignment>;
```

Note that I used `roleDefinitionId` first, then `principalId`, since I believe that the former one has a smaller scope than the later one, so I'm trying to go from micro to macro, scope-wise. Does this make sense? I can change it, I won't push back, but this is my argument for my current approach.

Wait, what other methods have these as the required parameters?

Reply:
The only methods that have these  properties in .Net are: CreateRoleAssignment, and CreateRoleAssignmentAsync.